### ISSUE TITLE
Custom Overlays

### ISSUE THREAD

**Issue by Hawkeye4040** [brandon-ray/foxhole-facility-planner#46]
I would love to see the ability to add layer(s) designated as user overlays. Typically they'd include additional texts, arrows, etc. Stuff that can really clutter up the screen. I will suggest it as another issue/feature suggestion but it's closer related and that's layer visibility. This would allow me to include a bunch of text, arrows, etc., and turn them off/on as desired to export screenshots neatly.

I think it would benefit that rather than simply making this just one layer allow users to create their own custom layer definitions. I've already made my own fork here I can work on it off to the side if this is something you'd like to see. I thought I'd bring it up not only as general courtesy/common sense but also just to check and see that this effort wasn't already underway.

I think it's a great and useful idea to add to what you've done here which I absolutely love by the way. This made it so I can actually tolerate running logi in Foxhole.

I can elaborate more or add some reference images/mockups but I think that's a good start to the idea.

**Comment by jimdcunningham** [comment]
Yeah, I'd be happy to hear more, concepts / mockups definitely help. I would definitely try out the dev site though haha! https://dev.foxholeplanner.com
